practice-one/src/Components/Common/productCard.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Card from './Card';
+
+const MyCard = ({

```

Comment:
what is MyCard?

Reply:
Done, change name MyCard into ProductCard